src/Bicep.Core.Samples/Files/AKS_LF/main.diagnostics.bicep

```diff
@@ -20,7 +20,7 @@ param osDiskSizeGB int = 0
 @minValue(1)
 @maxValue(50)
 param agentCount int = 3
-//@[6:16) [no-unused-params (Warning)] Parameter is declared but never used.\nSee https://aka.ms/bicep/linter/no-unused-params |agentCount|
+//@[6:16) [no-unused-params (Warning)] Parameter is declared but never used. bicep core(https://aka.ms/bicep/linter/no-unused-params)|agentCount|
```

Comment:
What's with the `bicep core(` in this message?

Reply:
"bicep core" is a combination of the Language Identifier and the analyzer name - the source of the information in this case is the linter whose analyzer name is core

If this feature is used for the Compiler or any regular Diagnostic, Source is initialized to LanguageIdentifier only